libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ChartConfigurationFlyoutProps.tsx

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { ErrorCohort } from "@responsible-ai/core-ui";
+
+export interface IChartConfigurationFlyoutProps {
```

Comment:
nit: both props and state interface can be in single file

Reply:
or can be moved to utils file too

Reply:
we have 24 files that are props only:
https://github.com/search?q=repo%3Amicrosoft%2Fresponsible-ai-toolbox+Props.ts&type=code
and we have 12 files that are state only:
https://github.com/search?q=repo%3Amicrosoft%2Fresponsible-ai-toolbox+State.ts&type=code
I hope it's ok to keep to this convention?  I just wanted to bring the line count down and this was the only thing I could think of.  Seems it needs deeper refactoring to bring it below the suggested limit of 300 lines.